src/templates/en/index.html

```diff
@@ -69,11 +70,50 @@ <h2>
         </button>
       </div>
       <div class="intro-year">2019</div>
-      <div class="intro-image">
-        <img class="home-browser" src="/static/images/home-browser.png" />
-        <img class="camada-751" src="/static/images/camada-751.png" />
-        <img class="camada-759" src="/static/images/camada-759.png" />
-        <img class="camada-752" src="/static/images/camada-752.png" />
+      <div class="intro-image-container">
+        <div class="intro-image">
+          <img class="home-browser" src="/static/images/home-browser.png" />
+          <img class="camada-751" src="/static/images/camada-751.png" />
+          <img class="camada-759" src="/static/images/camada-759.png" />
+          <img class="camada-752" src="/static/images/camada-752.png" />
```

Comment:
The red character appears stretched out and misaligned with the background graphic

![image](https://user-images.githubusercontent.com/1120896/65059932-e422cc00-d944-11e9-8355-ba0e20eb27b7.png)


Would it be easier if we flatten the browser graphic with the three characters into a single image? Or is there an advantage to having them separate for desktop/mobile layout?

Reply:
Yup, that would be easier 😄 The only difference between the desktop and mobile layout is that their sizes differ.

So either we have 1 image that can be scaled nicely for both desktop and mobile layouts or 2 images (one optimized for desktop and one optimized for mobile). WDYT?